nbs/13_questionnaire_handler.ipynb

```diff
@@ -185,7 +227,9 @@
     "    coding = dict(zip(code_df[code_from].astype(str), code_df[code_to]))\n",
     "    \n",
     "    \n",
-    "    if dict_df.loc[tab_field_name]['field_type'].strip() == 'Categorical (multiple)':\n",
+    "    field_type =  dict_df.loc[tab_field_name]['field_type']\n",
+    "    #if tab field is in 2 features sets it will be a series so just check the first case\n",
+    "    if isinstance(field_type, pd.Series) and field_type.iloc[0].strip() == 'Categorical (multiple)' or isinstance(field_type, str) and field_type.strip() == 'Categorical (multiple)':\n",
```

Comment:
we dont need the strip()
its the data owner responsibility to have only valid properties

Reply:
Also, If I understand correctly we have here a mix of use cases:
1.  if isinstance(field_type, pd.Series) - we need to check that it the same value for both and then--> field_type =   field_type.iloc[0] if not return original answer with a proper warning
2. field_type.strip() == 'Categorical (multiple)' - so the if actual only this part

Also I think this is the proper place to handle the case of multiple columns with the same name and not in the function "transform_dataframe"

Reply:
cool break up logic

Reply:
maybe remove from transform dataframe